docs/source/extensions/extension_visualization.rst

```diff
@@ -144,30 +144,90 @@ smoothing algorithm.
 .. figure:: https://raw.githubusercontent.com/asreview/asreview-visualization/master/docs/progression.png
    :alt: Progression
 
-API
----
 
-To make use of the more advanced features, you can also use the
-visualization package as a library. The advantage is that you can make
-more reproducible plots where text, etc. is in the place *you* want it.
-Examples can be found in module ``asreviewcontrib.visualization.quick``.
-Those are the scripts that are used for the command line interface.
+Plotting API
+------------
+
+To make use of the more advanced features and/or incorporate plotting 
+into code, you can use the visualization package as a library using the 
+build-in API.
+
+Setting up a plot
+~~~~~~~~~~~~~~~~~
+
+To set up a plot for a regerated h5 file (e.g. myreview.h5), this 
+code can be used:
 
 .. code:: python
 
     from asreviewcontrib.visualization.plot import Plot
 
-    with Plot.from_paths(["PATH_1", "PATH_2"]) as plot:
-        inc_plot = plot.new("inclusion")
-        inc_plot.set_grid()
-        inc_plot.set_xlim(0, 30)
-        inc_plot.set_ylim(0, 101)
-        inc_plot.set_legend()
+    with Plot.from_paths(["myreview.h5"]) as plot:
+        my_plot = plot.new(**kwargs)
         inc_plot.show()
-        inc_plot.save("SOME_FILE.png")
 
-Of course fill in ``PATH_1`` and ``PATH_2`` as the files you would like
-to plot.
 
-If the customization is not sufficient, you can also directly manipulate
-the ``self.ax`` and ``self.fig`` attributes of the plotting class.
+
+Multiple plots can be generated at the same time by adding the files to an array; ["myreview.h5", "myreview_2.h5"].
+
+\*\*kwargs has the following options:
+
+.. option:: plot_type="type"
+
+    *type* must be set to one or more of the available plot type; *inclusion*, *discovery*, *limit*, *progression*.
+
+.. option:: result_format="plot_format"
+
+    Can be set to "number" for absolute values or "percentage" (default) for percentages.
```

Comment:
Nice, have you considered using sphinx autodoc for this? Like in https://raw.githubusercontent.com/asreview/asreview/master/docs/source/API/reference.rst. This can simplify this part.